static/app/components/dropdownMenuItemV2.tsx

```diff
@@ -14,7 +14,12 @@ import overflowEllipsis from 'sentry/styles/overflowEllipsis';
 import space from 'sentry/styles/space';
 import {Theme} from 'sentry/utils/theme';
 
-type Priority = 'primary' | 'danger';
+/**
+ * Menu item priority. Currently there's only one option, but we may choose to
+ * add more in the future.
+ */
+type Priority = 'danger';
```

Comment:
Should we have a `default` priority?

Reply:
Hmm what would be the benefit of having one? `Button` has it but I've rarely ever seen it being used.

Reply:
IMO it's just more clear that it's a default, it can be helpful when you have a ternary for the value 

```jsx
// This
type={something ? 'primary' : 'default'}

// vs this
type={something ? 'primary' : undefined}
```

Just feels slightly better I think

Reply:
Ahh sounds good, added in https://github.com/getsentry/sentry/pull/34206/commits/28c8f21fc208158b89cef9f23a8eb768d070d96b!